addons/tutorial_03_multi_hop/manifest.json

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
Are we still implementing the "Rollback to original servers" at the end of this tutorial?

Reply:
yes, we do.

Reply:
Hmmmm - just ran this locally and I remained on Vienna -> Melbourne after the tutorial ended.

Reply:
ah! multi-hop sorry. I think we are OK with this.